index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ebuka Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <script src="https://kit.fontawesome.com/749225044f.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <div class="MainContainer">
+  <header>
+      <h3><a href="./index.html">My Logo</a></h3>
+      <nav>
+        <img src="./img/Menu.png" alt="Menu">
+      </nav>
+  </header>
+  
+  <section class="Profile">
+  <h1>I'm Ebuka, Glad to see you!</h1>
+  <p>I’m a software developer! I can help you build a product, 
+      feature or website Look through some of my work 
+      and experience! If you like what you see and have a project you
+       need coded, don’t hestiate to contact me.</p>
+       <span>LET’S CONNECT</span>
```

Comment:
- [ ] [OPTIONAL] The purpose of ```LET’S CONNECT``` is to call for action, so it should be a link, try to use ```<a>``` tag instead of ```<span>```